src/app/(sidebar)/account/muxed-create/page.tsx

```diff
@@ -165,6 +168,8 @@ export default function CreateMuxedAccount() {
         title="Muxed accounts are uncommon"
       >
         Don’t use in a production environment unless you know what you’re doing.
+        Read more about Muxed accounts{" "}
+        <Link href="https://developers.stellar.org/">here</Link>.
```

Comment:
Should we use a more specific link here (same on the parsed muxed account page)?

Reply:
+1. Maybe this link https://developers.stellar.org/docs/learn/encyclopedia/transactions-specialized/pooled-accounts-muxed-accounts-memos#muxed-accounts